CleverPush/CleverPush/Source/CleverPush.m

```diff
@@ -241,14 +241,14 @@ + (BOOL)hasSubscriptionAttributeValue:(NSString*)attributeId value:(NSString*)va
     return [self.CPSharedInstance hasSubscriptionAttributeValue:attributeId value:value];
 }
 
-+ (void)getAvailableTags:(void(^)(NSArray *))callback {
-    [self.CPSharedInstance getAvailableTags:^(NSArray *callbackInner) {
++ (void)getAvailableTags:(void(^)(NSArray <NSString*> *))callback {
+    [self.CPSharedInstance getAvailableTags:^(NSArray <NSString*> *callbackInner) {
```

Comment:
Please change from NSString to CPChannelTag and check all other methods again so we are 100% sure